cylc/flow/cfgspec/globalcfg.py

```diff
@@ -64,28 +66,28 @@
 
         .. versionchanged:: 8.0.0
 
-           This item was previously called ``startup handler``.
+           {}``startup handler``.
 
-        '''
+        '''.format(REPLACES)
```

Comment:
I think that I'd like everyone to weigh in. "This was previously called" is clearer, but less precise. I'd like to have opinion from @datamel and @hjoliver .

Reply:
Maybe change `cylc 7` to `Cylc 7` for tidiness? 
Could go for something even more precise like...`renamed to..`, since it is a renaming rather than a replacement?

Reply:
Don't necessarily need the mention of Cylc 7 because the admonition states it's changed in 8.0.0. But my main issue is with
> replaces the configuration X

I think the present tense, and the use of the word "configuration" to refer to a single item instead of the whole configuration, are somewhat confusing

Reply:
Happy with logic. WIll change